src/shared/util.js

```diff
@@ -2324,6 +2337,7 @@ exports.StatTimer = StatTimer;
 exports.StreamType = StreamType;
 exports.TextRenderingMode = TextRenderingMode;
 exports.UnexpectedResponseException = UnexpectedResponseException;
+exports.InvalidHeaderException = InvalidHeaderException;
```

Comment:
Remove `InvalidHeaderException`. The parser logic is an internal implementation detail and we don't need to expose the exception to external API users.